.travis.yml

```diff
@@ -6,8 +6,8 @@ branches:
     - rewrite
 node_js:
   - "node"
```

Comment:
At the moment setting "node" and '8' is the same https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions

Reply:
Are you sure? The `node` travis build shows v9.6.1.

See https://github.com/nodejs/Release#release-schedule.